mitodl/open-discussions
# Set imagekit prefix to ""

#### Pre-Flight checklist

#### What are the relevant tickets?
N/A

#### What's this PR do?
Fixes access to imagekit-generated avatar images via cloudfront by setting the imagekit cache prefix to "" instead of "CACHED/images"

#### How should this be manually tested?
- Set `AWS*` and `CLOUDFRONT_DIST` .env settings to the same as RC.
- Set `OPEN_DISCUSSIONS_USE_S3=True` in your .env
- In django admin, upload an avatar image for one of your fields.
- Go to the field page, you should see the avatar image, and it should have a cloudfront url.
